index.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const Bitmap = require(`${__dirname}/model/bitmap-constructor.js`);
+const ColorTransform = require(`${__dirname}/model/color-constructor.js`);
+const fileHelper = require(`${__dirname}/lib/bitmap-file-helper.js`);
+
+module.exports = exports = {};
+
+exports.invertBitmap = () => {
+  let onRead = (err, data) => {
```

Comment:
This seems like an unnecessary wrapper. Would be more straight forward to have the file helper at top and then could define function. That would flow easier for later reading.